usr/share/rear/backup/TSM/default/500_make_TSM_backup.sh

```diff
@@ -0,0 +1,29 @@
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+# Exit if TSM binary cannot be found.
+has_binary dsmc || Error "Can't find TSM client dsmc; Please check your configuration."
+
+# If the TSM client is found, do an incremental backup:
+backup_tsm_log=/var/lib/rear/backup_tsm_log
+
+if [[ ! -d "$backup_tsm_log" ]]; then
+    mkdir -p $v $backup_tsm_log
+fi
+
+LogPrint ""
+LogPrint "Starting Backup with TSM"
+LC_ALL=${LANG_RECOVER} dsmc incremental\
```

Comment:
Why `incremental` and not `save`? Could there be other uses who would *not* want to have `incremental` here?